cmd/neofs-node/morph.go

```diff
@@ -133,7 +133,7 @@ func makeNotaryDeposit(c *cfg) (util.Uint256, error) {
 
 		// gasDivisor defines what part of GAS balance (1/gasDivisor)
 		// should be transferred to the notary service
-		gasDivisor = 2
+		gasDivisor = 3
```

Comment:
What about all other, more frequent cases?

Reply:
I don't think we need this hotfix in master -- it "fixes" a rather peculiar and artificial scenario.
Let's discuss and fix it properly in another task.

Reply:
Hm, `2` is still a magic value anyway cause nobody knows what exact value is perfect for a deposit but it certainly does not allow making two deposits (caused for any reason) in a row

dropped